confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/RemoveDataNodeProcedure.java

```diff
@@ -77,8 +77,8 @@ protected Flow executeFromState(ConfigNodeProcedureEnv env, RemoveDataNodeState
           setNextState(RemoveDataNodeState.STOP_DATA_NODE);
           break;
         case STOP_DATA_NODE:
-          env.getDataNodeRemoveHandler().stopDataNode(tDataNodeLocation);
           env.getDataNodeRemoveHandler().removeDataNodePersistence(tDataNodeLocation);
```

Comment:
![image](https://user-images.githubusercontent.com/6756545/183865446-83efcbd0-96c0-4260-a78a-cdc13430fdcb.png)

I think the naming of disabledDataNode (in `DataNodeRemoveHandler.java`) is cool, rather than originalDataNode defined in `RegionMigrateProcedure.java`

Reply:
ok, done